returnn/datasets/basic.py

```diff
@@ -1374,16 +1394,26 @@ class DatasetSeq:
     Encapsulates all data for one sequence.
     """
 
-    def __init__(self, seq_idx, features, targets=None, seq_tag=None):
+    def __init__(
+        self,
+        seq_idx: int,
+        features,
+        targets=None,
+        seq_tag: Optional[str] = None,
+        epoch_continuous: Optional[float] = None,
```

Comment:
I'm a bit hesitating whether this really needs to be part of `DatasetSeq`...

Reply:
They don't /have/ to be extended, they can also choose to define `num_seqs`... But I agree having this distinction seems a bit off.